biojava-structure/src/main/java/org/biojava/nbio/structure/scop/ScopInstallation.java

```diff
@@ -913,10 +913,19 @@ private void initScopURLs() {
 		// first, try default scop
 		ScopMirror primary = new ScopMirror();
 		// If unreachable, try alternate Berkeley location
-		ScopMirror alt = new ScopMirror(
-				SCOP_DOWNLOAD_ALTERNATE,
-				"dir.cla.scop.%s.txt","dir.des.scop.%s.txt",
-				"dir.hie.scop.%s.txt","dir.com.scop.%s.txt");
+		ScopMirror alt;
+		if (scopVersion.startsWith("2.")) {
+			alt = new ScopMirror(
+					SCOP_DOWNLOAD_ALTERNATE,
+					"dir.cla.scope.%s.txt","dir.des.scope.%s.txt",
+					"dir.hie.scope.%s.txt","dir.com.scope.%s.txt");
+		}
+		else {
+			alt = new ScopMirror(
+					SCOP_DOWNLOAD_ALTERNATE,
+					"dir.cla.scop.%s.txt","dir.des.scop.%s.txt",
+					"dir.hie.scop.%s.txt","dir.com.scop.%s.txt");
+		}
```

Comment:
The 2 blocks in the if/else look identical. What was the intention of this change?

Reply:
One is *scope* while the other is *scop*.

Reply:
Thanks! sorry I missed that.